src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -50,27 +50,37 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
         }
 
         BenefitType benefitType = callback.getCaseDetails().getCaseData().getAppeal().getBenefitType();
+        Benefit benefit = Benefit.getBenefitByCodeOrThrowException(benefitType.getCode());
```

Comment:
I think maybe you can just call `callback.getCaseDetails().getCaseData().getBenefitType()` here and it will give you what you need.

Reply:
![image](https://user-images.githubusercontent.com/93123095/150180310-9df4b522-b8eb-47fc-b801-873928365b80.png)
the BenefitType do not contains benefit type information, it only have code and description. Unless update the BenefitType class?

Reply:
I meant in SscsCaseData, there is a `getBenefitType` method